pkg/config/environment_detection.go

```diff
@@ -127,6 +129,9 @@ func detectContainerFeatures() {
 
 	if IsKubernetes() {
 		detectedFeatures[Kubernetes] = struct{}{}
+		if Datadog.GetBool("orchestrator_explorer.enabled") {
```

Comment:
Does this mean if we are running kube and have the above setting set to true, the orchestrator explorer will collect on the leader. 

If we want this to be run on CLC runners instead we would need to configure the appropriate settings?

Reply:
if we are running in kube and the feature is activated it will inject a special _orchestrator autodiscovery service that will match this default config https://github.com/DataDog/datadog-agent/blob/1e00079b02c20392de505e2c6b711170cabf2956/Dockerfiles/cluster-agent/conf.d/orchestrator.d/conf.yaml.default (the default config can be overridden by any manual config)